pkg/pb/Backend.cs

```diff
@@ -0,0 +1,833 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: api/backend.proto
+// </auto-generated>
+#pragma warning disable 1591, 0612, 3021
+#region Designer generated code
+
+using pb = global::Google.Protobuf;
+using pbc = global::Google.Protobuf.Collections;
+using pbr = global::Google.Protobuf.Reflection;
+using scg = global::System.Collections.Generic;
+namespace Api {
```

Comment:
`Api` isn't a particularly friendly namespace. If we're going to check in generation (under the pretense of publishing a client package) consider adding `option csharp_namespace = "OpenMatch";`